### ISSUE TITLE
Proxy Model Support

### ISSUE THREAD

**Issue by LeonardoGentile** [django-polymorphic/django-polymorphic#351]
Hello,
in the doc introduction it says: 

> Support for proxy models.

but then it's the only place where proxies are mentioned, or maybe I did not look carefully. 

It's not clear to me if this package uses multi-table inheritance or not. In that case would be always true even when we want to establish inheritance using Proxies models only? In this case of course all the sub-types will have the same fields, only methods would change.

I've posted a more detail [question on SO](https://stackoverflow.com/questions/50525301/django-inheritance-and-polymorphism-with-proxy-models?noredirect=1#comment88064445_50525301) on the argument.

In short: is this package capable of providing this functionality?
Given a model A and its children **proxy** models B and C, can we get back the specific subtype B and C from the generic queryset on A?

Meaning, given `A.objects.all()` can weget back the specific subtype `B` and `C` without the extra DB queries or multi-table inheritance?

**Comment by vdboor** [comment]
Yes, you'll get the specific sub type.

The support comes from code that checks for `ContentType.objects.get_for_model(.., for_concrete_model=False)` This returns the content type of the proxy model instead of the concrete model. The code also checks for such situations and recasts the object dynamically.

**Comment by LeonardoGentile** [comment]
thanks 👍 
Is it highlighted somewhere in the doc?
